src/plugins/data/public/ui/search_bar/search_bar.tsx

```diff
@@ -95,6 +101,7 @@ export interface SearchBarOwnProps {
   onRefresh?: (payload: { dateRange: TimeRange }) => void;
   indicateNoData?: boolean;
   queryStatus?: QueryStatus;
+  queryStringManager: QueryStringManager;
```

Comment:
Shall we need to change `QueryStringManager` to `QueryStringContract`? Since `data.query.queryString` is a `QueryStringContract` instead of `QueryStringManager`. It will show a type error in  `src/plugins/data/public/ui/search_bar/create_search_bar.tsx` like screenshot below.
![image](https://github.com/user-attachments/assets/5fa9c144-8986-491f-a5ef-4053e4f5c309)